### ISSUE TITLE
Registration dates and web site dates are in conflict

### ISSUE THREAD

**Issue by timlinux** [foss4g2018/foss4g2018#97]
I had a query from an attendee:

> I’m a bit confused with the accommodation timeline. In FOSS4G website, they only mentioned that registration fee includes accommodation from 28 to 31. But FOSS4G happens from 27th August to 2 September (includes Workshops and code sprints). Do we have to arrange for extra accommodations for that extra days, or we only have to be present in 28th to 31 only?

@brompo @markiliffe can we clarify if this is correct?

Our website dates are:

27, 28 AUGUST 2018 : WORKSHOPS
29, 30, 31 AUGUST 2018 : THREE DAY MAIN CONFERENCE
1, 2 SEPTEMBER 2018 : CODE SPRINTS AND COMMUNITY EVENTS

So from my understanding the accommodation covers the days of the main conference only. I think we need to:

* make it very clear on registration that the included accom is only for the main conference
* offer a way for attendees to book the additional days if they are staying for workshops / sprints (or instructions on where to go and do it)
* notify those who have already registered that this is the case

**Comment by timlinux** [comment]
cc @lucernae

**Comment by markiliffe** [comment]
Yes, the included accommodation is only for the days of the main conference track - I'll leave it to the website team to create the copy and make this clearer. Thanks for bringing it to my attention though @timlinux ! :-)
